packages/platform-feeds/index.js

```diff
@@ -257,10 +246,10 @@ function buildFeedObject(dateNum, item) {
     date: item.date,
     datenum: dateNum,
     tags: item.categories,
-    text: item.summary,
-    html: item.summary,
-    brief_text: item.description,
-    brief_html: item.description,
+    brief_text: item.summary,
+    brief_html: item.summary,
```

Comment:
Why are these two different keys, when they both use the same data? The `brief_text` contains HTML in all the feeds I tried with the example, and it's usually also the same as the full HTML (i.e. not brief).